Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -754,15 +754,12 @@ TextPosition TextDocument::first_word_break_before(TextPosition const& position,
 
     target.set_column(target.column() - modifier);
```

Comment:
The commit title is a bit goofy, maybe something like this?
`LibGUI: Prevent crashes/hangs when deleting words backwards`

Commit message:

`[..] where the previous content with format [..]` -> `[..] where the previous content has the format [..]`

`downflow` -> `underflow`

`a invalid` -> `an invalid`

Reply:
Sorry for my poor English! Commit message has been fixed.